db/schema.rb

```diff
@@ -10,55 +10,55 @@
+    t.string 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'post_id', null: false
+    t.bigint 'author_id', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['post_id'], name: 'index_comments_on_post_id'
   end
 
-  create_table "likes", force: :cascade do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.bigint "post_id", null: false
-    t.bigint "author_id", null: false
-    t.index ["author_id"], name: "index_likes_on_author_id"
-    t.index ["post_id"], name: "index_likes_on_post_id"
+  create_table 'likes', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'post_id', null: false
+    t.bigint 'author_id', null: false
+    t.index ['author_id'], name: 'index_likes_on_author_id'
+    t.index ['post_id'], name: 'index_likes_on_post_id'
   end
 
-  create_table "posts", force: :cascade do |t|
-    t.string "title"
-    t.string "text"
-    t.integer "comments_counter"
-    t.integer "likes_counter"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.bigint "author_id", null: false
-    t.index ["author_id"], name: "index_posts_on_author_id"
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.string 'text'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id', null: false
+    t.index ['author_id'], name: 'index_posts_on_author_id'
   end
 
-  create_table "users", force: :cascade do |t|
-    t.string "name"
-    t.string "photo"
-    t.string "bio"
-    t.integer "posts_counter"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.string 'bio'
+    t.integer 'posts_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
   end
 
-  add_foreign_key "comments", "posts"
-  add_foreign_key "comments", "users", column: "author_id"
-  add_foreign_key "comments", "users", column: "author_id"
-  add_foreign_key "likes", "posts"
-  add_foreign_key "likes", "users", column: "author_id"
-  add_foreign_key "likes", "users", column: "author_id"
-  add_foreign_key "posts", "users", column: "author_id"
-  add_foreign_key "posts", "users", column: "author_id"
+  add_foreign_key 'comments', 'posts'
+  add_foreign_key 'comments', 'users', column: 'author_id'
+  add_foreign_key 'comments', 'users', column: 'author_id'
+  add_foreign_key 'likes', 'posts'
+  add_foreign_key 'likes', 'users', column: 'author_id'
+  add_foreign_key 'likes', 'users', column: 'author_id'
+  add_foreign_key 'posts', 'users', column: 'author_id'
+  add_foreign_key 'posts', 'users', column: 'author_id'
```

Comment:
- [OPTIONAL] When I tried to setup the database I got an error that there are DuplicateObject, please kindly update your `schema.rb` file and delete the duplicated objects 🙏🏾